src/app/dataset-view/additional-components/dataset-settings-component/tabs/compacting/dataset-settings-compacting-tab.component.html

```diff
@@ -0,0 +1,88 @@
+<div class="w-75 content-container">
+    <h2>Compacting</h2>
+    <mat-divider class="mt-1 mb-2"></mat-divider>
+
+    <div class="box p-4 mt-3">
+        <h4 class="subtitle">Hard compaction</h4>
+        <form [formGroup]="hardCompactionForm">
+            <div class="row">
+                <div class="col-md-3">
+                    <div class="d-flex align-items-center pt-1">
+                        <app-tooltip-icon class="mt-1" [tooltip]="MAX_SLICE_SIZE_TOOLTIP"></app-tooltip-icon>
+                        <span class="d-inline-block">Max slice size:</span>
+                    </div>
+                </div>
+                <div class="col-md-9 d-flex">
+                    <select class="form-select me-4 form-control-width" formControlName="sliceUnit">
+                        <option [value]="SliceUnit.B">B</option>
+                        <option [value]="SliceUnit.KB">KB</option>
+                        <option [value]="SliceUnit.MB">MB</option>
+                        <option [value]="SliceUnit.GB">GB</option>
+                    </select>
+                    <input type="number" min="1" class="form-control form-control-width" formControlName="sliceSize" />
+                    <ng-container
+                        *ngIf="
+                            hardCompactionForm?.get('sliceSize')?.errors?.minNumber &&
+                            (hardCompactionForm.get('sliceSize')?.touched || hardCompactionForm.get('sliceSize')?.dirty)
```

Comment:
It's usually cleaner to provide a component property that returns a particular form control.

Reply:
Fixed